ResourceManager/GitExtensionsControl.cs

```diff
@@ -37,6 +39,8 @@ protected override void OnLoad(EventArgs e)
         {
             base.OnLoad(e);
 
+            _loaded = true;
```

Comment:
This should be on line:48.

Reply:
or be renamed _hotkeysLoaded, that is how the value is used.